.buildbot_dockerfile_debian

```diff
@@ -3,6 +3,7 @@ ARG CI_UID
 RUN useradd -m -u ${CI_UID} ci
 RUN apt-get update && \
 apt-get -y install build-essential curl procps file
```

Comment:
I think you need to end this line with `\` and remove `apt-get -y install` from the following line. [I remember this format being fussy, and I *think* that's what works.]

Reply:
Doh, I see it now.

should be fixed in [949e713](https://github.com/softdevteam/grmtools/pull/527/commits/949e71328c59ca1a1ec9b8d1b5e09e7f2e8762b0)